Loop/Floodgate.swift

```diff
@@ -40,7 +41,25 @@ final class Floodgate<State, Event>: FeedbackEventConsumer<Event> {
         dispose()
     }
 
-    func bootstrap(with feedbacks: [FeedbackLoop<State, Event>.Feedback]) {
+    func bootstrap(with feedbacks: [Loop<State, Event>.Feedback]) {
+        self.feedbacks = feedbacks
+
+        for feedback in feedbacks {
+            // Pass `producer` which has replay-1 semantic.
+            feedbackDisposables += feedback.events(producer, self)
+        }
+
+        reducerLock.perform {
+            drainEvents()
+        }
+    }
+
+    func unplugFeedbacks() {
+        feedbackDisposables.dispose()
+        feedbackDisposables = CompositeDisposable()
+    }
+
+    func replugFeedbacks() {
```

Comment:
thought about simply invoking `bootstrap(with:)` again from the `Loop`, but decided that the `Floodgate` should be the one retaining the feedbacks. It also allowed having a symmetrical API (unplug/replug)

However the `Loop` being the one retaining the feedbacks also seems like a valid approach.